src/views/DetailView.js

```diff
@@ -1,14 +1,15 @@
-import { VivViewerLayer, StaticImageLayer } from '../layers';
+import { VivViewerLayer, StaticImageLayer, ScaleBarLayer } from '../layers';
 import VivView from './VivView';
 import { getVivId } from './utils';
 
 /**
  * This class generates a VivViewerLayer and a view for use in the VivViewer as a detailed view.
  * */
 export default class DetailView extends VivView {
-  getLayers({ props }) {
+  getLayers({ props, viewStates }) {
     const { loader } = props;
     const { id } = this;
+    const thisViewState = viewStates[id];
```

Comment:
perhaps `layerViewState` or `viewStateUpdate` or `viewState`? Confusing to prefix with this in javascript because it is a keyword...

Reply:
Makes sense